### PR TITLE
Document possible vulnerabilities for the druid-ranger-security

### PR DESCRIPTION
**Pull request by bolkedebruin**
In certain configurations the ranger plugin can expose vulnerabilities due
to some of its dependencies having CVEs.

@ccaominh @himanshug I have chosen to document rather than exclude. For log4j it seems not to be a full drop in replacement. In both it will only manifest itself in certain configurations which are non default.

In the mean time I am pushing Ranger to have a new release which will remove these issues.

### CODE REVIEW

File: owasp-dependency-check-suppressions.xml [hunk 1]
```diff
@@ -187,4 +187,36 @@
      <packageUrl regex="true">^pkg:npm/node\-sass@.*$</packageUrl>
      <vulnerabilityName>CWE-400: Uncontrolled Resource Consumption ('Resource Exhaustion')</vulnerabilityName>
   </suppress>
+  <suppress>
+    <!--
+      ~ TODO: Fix when Apache Ranger 2.1 is released
```
Comment (himanshug) [inline owasp-dependency-check-suppressions.xml:192]: can you create a github issue for this ?

**Comment by himanshug** [comment]
LGTM , can you verify that the build gets fixed.

**Comment by bolkedebruin** [comment]
I did a `mvn clean && mvn install -DskipTests=true -Dtar -T 8` and a `mvn dependency-check:check` both passed locally.

File: owasp-dependency-check-suppressions.xml (hunk 1 above)
Comment (bolkedebruin) [reply owasp-dependency-check-suppressions.xml:192]: Done

**Comment by himanshug** [comment]
sgtm, but I will let @ccaominh approve this as he is more familiar with the specifics.

**Pull request merged**